Rakefile

```diff
@@ -147,6 +130,7 @@ namespace :cross_agent_tests do
   end
 end
 
+desc 'Start Pry'
```

Comment:
In these situations, I have a preference for using generic terminology as much as possible to avoid single product/implementation lock in. Perhaps:

```diff
- desc 'Start Pry'
+ desc 'Start an interactive console session'
```

Reply:
Great great call. [Addressed](https://github.com/newrelic/newrelic-ruby-agent/pull/1421/commits/e3c43a60bacc3b12f6541cd95d0aab5b0d2ee276)!